external/storm-hive/src/main/java/org/apache/storm/hive/bolt/HiveBolt.java

```diff
@@ -134,22 +131,16 @@ public void execute(Tuple tuple) {
                     collector.ack(t);
                 tupleBatch.clear();
             }
+        } catch(SerializationError se) {
+            LOG.info("Serialization exception occurred, tuple is acknowledged but not written to Hive.", tuple);
+            collector.ack(tuple);
```

Comment:
I am fine with it how the code is, like I said my comments were all minor.  I was just thinking that if there are issues with the serialization I as a user am not going to be looking at the logs for errors, I am going to be looking at metrics, like failed tuples.  Perhaps we could have a follow on JIRA to add an IMetric for the number of tuples lost because of serialization errors.